README.md

```diff
@@ -102,6 +105,7 @@ Runs on [Agoric](https://github.com/agoric/)
 [SesComputingGuide]: https://github.com/endojs/endo/blob/master/packages/ses/docs/secure-coding-guide.md
 
 [PolicyDoc]: ./docs/policy.md
+[ScuttlingDoc]: ./docs/scuttling.md
 [LavamoatNode]: ./packages/node
 [LavamoatBrowserify]: ./packages/browserify
 [LavamoatViz]: ./packages/viz
```

Comment:
nit: for some reason i can't see/check these links (unless its maybe github metadata)

<img width="527" alt="Screenshot 2023-03-28 at 17 32 43" src="https://user-images.githubusercontent.com/1881059/228307874-82539533-6ba6-4df4-a20d-f0117402c4a9.png">

but not related to this PR